model.py

```diff
@@ -361,7 +363,6 @@ def forward_loss(self, img_emb, cap_emb, **kwargs):
         """Compute the loss given pairs of image and caption embeddings
         """
         loss = self.criterion(img_emb, cap_emb)
-        self.logger.update('Le', loss.data[0], img_emb.size(0))
```

Comment:
Can you comment on why this line is removed? Thanks.

Reply:
I am not sure at this moment, I guess there is no need for it

Reply:
This line is important for analyzing the behaviour of training as it logs the loss. Would you be able to add it back? The rest of the PR looks good to me. Thanks.

Reply:
Done!